src/Components/SavedSearchAlert/SavedSearchAlertModal.tsx

```diff
@@ -97,16 +173,53 @@ export const SavedSearchAlertModal: FC<SavedSearchAlertFormProps> = ({
       const result = {
         id: response.createSavedSearch?.savedSearchOrErrors.internalID!,
       }
+
+      if (values.hearFromArtsyAdvisor) {
+        await createAdvisoryOpportunity(relayEnvironment, {
+          searchCriteriaID: result.id,
+          message: values.message,
+          phoneNumber: values.phoneNumber,
+          phoneCountryCode: values.phoneCountryCode,
+        })
+      }
       onCreateAlert?.(result)
     } catch (error) {
       logger.error(error)
     }
   }
 
   return (
-    <Formik<SavedSearchAlertFormValues>
-      initialValues={initialValues}
+    <Formik<SavedSearchAlertFormValues & HearFromArtsyAdvisorFormValues>
+      initialValues={{
+        message: "",
+        hearFromArtsyAdvisor: false,
+        phoneNumber: phone,
+        phoneCountryCode: regionCode,
+        ...initialValues,
+      }}
+      enableReinitialize
       onSubmit={handleSubmit}
+      validationSchema={Yup.object().shape({
+        phoneNumber: Yup.string()
+          .test({
+            name: "phone-number-is-valid",
+            message: "Please enter a valid phone number",
+            test: (national, context) => {
+              if (!context.parent.hearFromArtsyAdvisor) return true
```

Comment:
If the advisor checkbox isn't checked, we can always 'validate' this field.

Otherwise, it cannot be blank, and must be valid based on the client-side check to MP.